### PR TITLE
LPS-100424 References from deleted attachments cannot be removed in edit mode in Chrome

### PR DESCRIPTION
**Pull request by MarcellGyopos**
Hi @adolfopa 

I realized that my previous solution is incomplete and may cause unexpected behavior.
Could you please review this pull request?

Thank you and best regards,
Marcell

### CODE REVIEW

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js [hunk 1]
```diff
@@ -343,12 +343,12 @@
...
-						r[2] === undefined
-							? options && options.defaultImageText
-								? options.defaultImageText
-								: ''
-							: r[2].replace(/~(.)/g, '$1');
+					if(r[2] != undefined){
+						img.alt = r[2].replace(/~(.)/g, '$1');
+					}
+					else if( options && options.defaultImageText) {
+						img.alt = opstions.defaultImageText
```
Comment (adolfopa) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js]: opstions == options

Spacing should be consistent with the rest of the code (there's a blank just after the open paren)

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js [hunk 2]
```diff
@@ -343,12 +343,12 @@
 
 					var img = document.createElement('img');
 					img.src = imagePath;
-					img.alt =
-						r[2] === undefined
-							? options && options.defaultImageText
-								? options.defaultImageText
-								: ''
-							: r[2].replace(/~(.)/g, '$1');
+					if(r[2] != undefined){
```
Comment (adolfopa) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js]: wouldn't `if (r[2]) {` be enough?

Comment (MarcellGyopos) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js]: I'm not a javascript guru, I left the previous condition, just negated it.

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js (hunk 1 above)
Comment (MarcellGyopos) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/creole/creole_parser.js]: Thanks for reviewing, I'll fix it based on your comments and send you a new pull request.

**Pull request closed without merging**